templates/partners/desktop.html

```diff
@@ -6,144 +6,245 @@
-        <p>Canonical collaborates with the world&rsquo;s leading vendors to optimise and certify Ubuntu on 100s of laptops, desktops and workstations for a secure and constantly updated platform.</p>
-        <p>Whether built with the developer in mind or to power AI ambitions, these out-of the-box platforms give practitioners everything they need to transform their enterprise IT.</p>
-        <a href="https://certification.ubuntu.com/desktop" class="p-button"><span>Find a certified PC</span></a><a href="/partners/become-a-partner" class="p-button--positive">Become a partner</a>
-      </div>
-      <div class="col-5 col-medium-2 u-vertically-center">
-        <ul class="p-logo-section__items">
-          {% for partner in partners %}
-          <li class="p-logo-section__item">
-            <img class="p-logo-section__logo" src="{{ partner.logo }}" style="min-width: 5rem;" alt="{{ partner.slug}}">
-          </li>
-          {% endfor %}
-        </ul>
-      </div>
+
+<div class="row">
+  <div class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
+    <ol class="p-breadcrumbs__items p-text--small-caps">
+      <li class="p-breadcrumbs__item "><a href="/partners">Partners</a></li>
+      <li class="p-breadcrumbs__item ">Desktop</li>
+    </ol>
+  </div>
+  <hr class="p-rule">
+</div>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <div class="col">
+      <h1>Desktop Programme</h1>
     </div>
-  </section>
-  <section class="p-strip--light is-deep">
-    <div class="row">
-      <div class="col-12 col-medium-6">
-        <h2 style="margin-bottom: 3rem;">Why buy a certified Ubuntu desktop?</h2>
-      </div>
-      <div class="col-4 col-medium-2 p-card--highlighted">
-        <h3 class="p-matrix__title">Preloaded with Ubuntu</h3>
-        <div class="p-matrix__desc">
+    <div class="col">
+      <p>Canonical collaborates with the world&rsquo;s leading vendors to optimise and certify Ubuntu on 100s of laptops, desktops and workstations for a secure and constantly updated platform.</p>
+      <p>Whether built with the developer in mind or to power AI ambitions, these out-of the-box platforms give practitioners everything they need to transform their enterprise IT.</p>
+      <a href="/partners/become-a-partner" class="p-button--positive">Become a partner</a>
+      <a href="https://certification.ubuntu.com/desktop" class="p-button"><span>Find a certified PC</span></a>
+    </div>
+  </div>
+</section>
+<section class="p-section">
+  <div class="row">
+    <hr class="p-rule">
+  </div>
+  <div class="row--50-50">
+    <div class="col">
+      <h2>A selection<br class=u-hide--small> of our Desktop partners</h2>
+    </div>
+    <div class="col">
+      <ul class="p-logo-section__items" style="justify-content: space-around;">
+        {% for partner in partners %}
+        <li class="p-logo-section__item">
+          <img class="p-logo-section__logo" src="{{ partner.logo }}" style="min-width: 5rem;" alt="{{ partner.slug}}">
```

Comment:
@ClementChaumel what is this for? Is it so cdns don't mangle the images? `style="min-width: 5rem;" `